app/src/main/java/com/willowtree/vocable/presets/CategoriesFragment.kt

```diff
@@ -59,9 +59,6 @@ class CategoriesFragment : BaseFragment<CategoriesFragmentBinding>() {
             with(categoryButton.root as CategoryButton) {
                 tag = category
                 text = localizedResourceUtility.getTextFromCategory(category)
-                action = {
```

Comment:
If we leave this in and the user taps on the category title, there will be a quick glitch of the phrases as it "refreshes" again and it looks a bit strange for the user. Happy to leave this in or comment it out though if we don't want to completely delete it out.

Reply:
Do we know why this was put in? Why we decided that we'd refresh the phrases when the user taps on the button? You can remove this if others thinks it's also okay 👍🏼 :3 but we should figure out how come we even put it in in the first place ;D

Reply:
Rudy (and Spud) pointed out that the tablet experience is quite different in that there are multiple categories visible at a time when a user scrolls forward/backward through the categories. Rudy pointed out that in iOS, the tablet and landscape experience requires the user to click on the category for the phrases to update.

Soooo...I put back this action callback so that the category titles can function as buttons 👍  :']